api-client/src/main/java/com/xing/api/data/profile/CareerLevel.java

```diff
@@ -15,8 +15,10 @@
  */
 package com.xing.api.data.profile;
 
+import com.xing.api.data.jobs.Job;
+
 /**
- * Possible values for the user's career level at a {@link Company}.
+ * Possible values for the user's career level at a {@link Company} or a {@link Job}.
```

Comment:
Hm... __checkstyle__ thinks `Job` is an unused import... Please change to `{@linkplain com.xing.api.data.jobs.Job Job}` this should make it quite.